.github/workflows/rocks.yml

```diff
@@ -0,0 +1,60 @@
+name: Rocks server
+on:
+  schedule:
+    - cron: '0 */3 * * *'
```

Comment:
This job is not heavy, let's run every hour.

Reply:
Done